Formula/amp.rb

```diff
@@ -0,0 +1,37 @@
+class Amp < Formula
+  desc "Text editor for your terminal"
+  homepage "https://amp.rs"
+  url "https://github.com/jmacdonald/amp/archive/0.3.2.tar.gz"
+  sha256 "9080c894111149d58a0b805e0c08edaf1d9e7cd56c4b955ba9111feec6b55208"
+  head "https://github.com/jmacdonald/amp.git"
+
+  depends_on "rust" => :build
+  depends_on "cmake" => :build
+  depends_on "openssl"
+  depends_on "libssh2"
```

Comment:
This doesn't seem to be getting linked.

Reply:
Getting an openssl dependency error why trying to run the build artifact locally, too:
`dyld: Library not loaded: @@HOMEBREW_PREFIX@@/opt/openssl/lib/libssl.1.0.0.dylib`

Odd, because the tests pass when I remove the `assert_match` line, so the app is launching correctly in CI.

Reply:
If you're downloading the bottle from CI, extracting it manually & trying to run that then that's expected. Homebrew doesn't really support doing that with bottles.

Reply:
@ilovezfs how were you able to see that `libssh2` isn't getting linked? I've looked through the build output and haven't found anything that hints at that. As far as I can tell, the only thing that's failing is the test phase itself; there's no other output that I'm seeing that points to a failed compile or link of any kind.

I'm admittedly a bit lost at this point. The formula installs locally without issue and the tests are all passing as well. I know compiling with Homebrew-provided `openssl` libraries requires setting flags, and I've added the appropriate `LDFLAGS`, `CPPFLAGS`, and `PKG_CONFIG_PATH` environment variables, but to no avail. How can I replicate the build environment to debug this? It's a bit of a black box at the moment. 🙁 

I would very much appreciate any additional help either of you can provide. 🙏

Reply:
`otool -L` or `brew linkage`.